storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Exceptions/ExceptionEx.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Dotnet.Storm.Adapter.Extensions
+{
+    public static class ExceptionEx
+    {
+        public static string GetMessageTrace(this Exception ex)
+        {
+            StringBuilder result = new StringBuilder();
```

Comment:
I think that ex.ToString() gives you what you want - see this: https://stackoverflow.com/questions/4272579/how-to-print-full-stack-trace-in-exception